src/cache-modified-extensions.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Cache modified extensions - documented at spec/amp-cache-guidelines.md under
+ * headline "Guidelines: Adding a new cache to the AMP ecosystem"
+ */
+const CACHE_MODIFIED_EXTENSIONS = ['amp-geo'];
+
+/**
+ * Get status of whether an extension is modified by the cache when served
+ * @param {string} extensionId
+ * @return {boolean}
+ */
+export function isCacheModifiedExtension(extensionId) {
+  return (
+    Boolean(extensionId) &&
```

Comment:
Boolean check shouldn't be necessary.

Reply:
Not necessary, but prevents an unnecessary loop if an empty string is passed for `extensionId`.